src/components/LocationCard/style.css

```
@@ -0,0 +1,19 @@
+.ant-card-bordered {
```

add .location-card before the name of class
```suggestion
.location-card .ant-card-bordered {
```